charts/cluster/values.yaml

```
@@ -127,6 +128,53 @@ recovery:
         name: ""
         key: ""
 
+  # See: https://cloudnative-pg.io/documentation/1.24/cloudnative-pg.v1/#postgresql-cnpg-io-v1-Import
+  import:
+    # -- One of `microservice` or `monolith.`
+    # See: https://cloudnative-pg.io/documentation/1.24/database_import/#how-it-works
+    type: "microservice"
+    # -- Databases to import
+    databases: []
+    # -- Roles to import
+    roles: []
+    # -- List of SQL queries to be executed as a superuser in the application database right after is imported.
```

```suggestion
    # -- List of SQL queries to be executed as a superuser in the application databases right after is imported.
```
Are the SQL queries are applied on one or all databases ?